### ISSUE TITLE
Rule change request: indent: allow "off" for "outerIIFEBody"

### ISSUE THREAD

**Issue by papb** [eslint/eslint#11377]
**What rule do you want to change?** [indent](https://eslint.org/docs/rules/indent)

**Does this change cause the rule to produce more or fewer warnings?** Fewer warnings when applied (but the default behavior would be the same)

**How will the change be implemented? (New option, new default behavior, etc.)?** New option, default behavior unchanged

Several options for the `indent` rule accept the string `"off"` to disable the check and let the user do it however they want. But a few of them do not, such as `"outerIIFEBody"`. I would like to be able to set `"outerIIFEBody"` to `"off"`. Currently I have to choose between 0 or 1 but both choices enforce something, and I don't want to enforce anything.

Also, I find it curious that several of the options support the `"off"` setting but only `"outerIIFEBody"` and a few others don't.

**Please provide some example code that this change will affect:**

```js
// File 1
(function() {
    a();
)();

// File 2
(function() {
b();
)();
```

**What does the rule currently do for this code?** One of the files will cause a warning, depending whether I choose 0 or 1 for the `"outerIIFEBody"` rule.

**What will the rule do after it's changed?** With the `"off"` option, eslint would not give a warning for any of those cases, while still checking the other indentation rules for the other options.

**Are you willing to submit a pull request to implement this change?** Sadly I don't have time for this at the moment...

**Comment by platinumazure** [comment]
I'll support this. Definitely makes sense to support turning off indentation enforcement in this area for consistency.

**Comment by papb** [comment]
What does it take for an issue to *"reach accepted status"*? Can I do something to help (except implementing the change, which I can't do right now sadly)?

Do you know any steps I can take, @platinumazure? Thanks for looking and sorry to bother.

**Comment by platinumazure** [comment]
@papb No worries. It looks like this just completely skipped through the cracks. I can ask some team members to see if this might be worth reopening and accepting.

As for our process for accepting issues: Please see [here](https://eslint.org/docs/maintainer-guide/issues#accepting-issues) for more details. Basically, we would need someone to champion the rule, and 3 others on the team to give it a :+1: and nobody to give it :-1:. I gave :+1:, but we had no champion and no other :+1:. I could potentially see myself championing this, but I don't have time to look into that just yet.

**Comment by papb** [comment]
@platinumazure - Thanks for looking into it :grimacing: I really appreciate it.

And thanks for the info as well :)

**Comment by platinumazure** [comment]
Reopening for one more evaluation attempt (will close in a week or so if I can't get support).

I'll champion this.

**Comment by kaicataldo** [comment]
@platinumazure Looks like it might be time to close this. Are you still championing this proposal?

**Comment by papb** [comment]
It's unfortunate, but if no one else is interested, I guess it makes sense to close :frowning_face:

**Comment by diegocr** [comment]
Don't get sad, the thumbs up in your initial message shows more people is interested, and count me on that as well.

ESLint team, please make this happen :)

**Comment by papb** [comment]
@diegocr Ah, thanks!! :smiley: I hadn't noticed the reactions to my first post!

**Comment by kaicataldo** [comment]
@eslint/eslint-team Looks like we just need one more :+1: to accept this.

**Comment by mdjermanovic** [comment]
Accepted now.
